DataManagementSystem/DB/FileCatalogComponents/DirectoryTreeBase.py

```diff
@@ -336,148 +337,156 @@ def getDirectoryParameters( self, path ):
 #####################################################################
   def _setDirectoryParameter( self, path, pname, pvalue ):
```

Comment:
In this method, you are using the actual table name explicitly, so I think you should not implement it here, but in the DirectoryLevelTree